index/generator/schema/schema.go

```diff
@@ -88,6 +88,17 @@ type Schema struct {
 	Git               *Git                   `yaml:"git,omitempty" json:"git,omitempty"`
 }
 
+// IndexType describes the type of index entry
+type IndexType string
+
+const (
+	// SampleIndexType represents a sample index
+	SampleIndexType IndexType = "sample"
```

Comment:
Update `SampleIndexType` to `SampleDevfileType` and the comment.

Reply:
updated!